core/controllers/topics_and_skills_dashboard.py

```diff
@@ -166,30 +166,19 @@ def get(self):
         self.render_json(self.values)
 
 
-class UnassignSkillDataHandler(base.BaseHandler):
-    """Provides data for Unassign skill modal."""
+class SkillAssignmentsHandler(base.BaseHandler):
+    """Retrieving information about which topics contain the given skill."""
```

Comment:
Retrieving information --> Provides information

(Don't blindly copy my comments, make sure they are appropriate for the context you're using them in. No other handler definition uses "...ing".)

Reply:
Ok will keep that in blind. 
Done.